routes/root.js

```diff
@@ -266,7 +265,7 @@ router.get('/dashboard/decks/:page', async (req, res) => {
   }
 });
 
-router.get('/landing', async (req, res) => {
+router.get('/landing', async (_, res) => {
```

Comment:
Can you remove all of these random changes?

Reply:
Yeah. I just had my IDE autofix all warnings it found like unused params and unneeded awaits.

Reply:
Can you turn off that setting and remove these changes?